Mathlib/Order/Lattice.lean

```diff
@@ -218,38 +218,38 @@ theorem sup_le_sup_right (h₁ : a ≤ b) (c) : a ⊔ c ≤ b ⊔ c :=
   sup_le_sup h₁ le_rfl
 #align sup_le_sup_right sup_le_sup_right
 
-theorem sup_idem : a ⊔ a = a := by simp
+lemma sup_idem (a : α) : a ⊔ a = a := by simp
 #align sup_idem sup_idem
 
-instance : Std.IdempotentOp (α := α) (· ⊔ ·) := ⟨@sup_idem _ _⟩
+instance : Std.IdempotentOp (α := α) (· ⊔ ·) := ⟨sup_idem⟩
 
-theorem sup_comm : a ⊔ b = b ⊔ a := by apply le_antisymm <;> simp
+lemma sup_comm (a b : α) : a ⊔ b = b ⊔ a := by apply le_antisymm <;> simp
 #align sup_comm sup_comm
 
-instance : Std.Commutative (α := α) (· ⊔ ·) := ⟨@sup_comm _ _⟩
+instance : Std.Commutative (α := α) (· ⊔ ·) := ⟨sup_comm⟩
 
-theorem sup_assoc : a ⊔ b ⊔ c = a ⊔ (b ⊔ c) :=
+lemma sup_assoc (a b c : α) : a ⊔ b ⊔ c = a ⊔ (b ⊔ c) :=
   eq_of_forall_ge_iff fun x => by simp only [sup_le_iff]; rw [and_assoc]
 #align sup_assoc sup_assoc
 
-instance : Std.Associative (α := α)  (· ⊔ ·) := ⟨@sup_assoc _ _⟩
+instance : Std.Associative (α := α)  (· ⊔ ·) := ⟨sup_assoc⟩
 
 theorem sup_left_right_swap (a b c : α) : a ⊔ b ⊔ c = c ⊔ b ⊔ a := by
-  rw [sup_comm, @sup_comm _ _ a, sup_assoc]
+  rw [sup_comm, sup_comm a, sup_assoc]
 #align sup_left_right_swap sup_left_right_swap
 
-theorem sup_left_idem : a ⊔ (a ⊔ b) = a ⊔ b := by simp
+lemma sup_left_idem (a b : α) : a ⊔ (a ⊔ b) = a ⊔ b := by simp
```

Comment:
I'd prefer not to adjust `lemma`/`theorem` as part of this diff, it just makes it noisy.

Reply:
This is kind of a pain to undo